Assets/LeapMotion/Core/Scripts/Attributes/Editor/CombinablePropertyDrawer.cs

```diff
@@ -23,8 +23,14 @@ public class CombinablePropertyDrawer : PropertyDrawer {
 
     private List<CombinablePropertyAttribute> attributes = new List<CombinablePropertyAttribute>();
     private void getAttributes(SerializedProperty property) {
-      if (!_cachedAttributes.TryGetValue(fieldInfo, out attributes)) {
-        attributes = new List<CombinablePropertyAttribute>();
+      GetAttributes(property, fieldInfo, ref attributes);
+    }
+
+    public static void GetAttributes(SerializedProperty property,
+                                     FieldInfo fieldInfo,
+                                     ref List<CombinablePropertyAttribute> outAttributes) {
```

Comment:
this can be an out property instead of a ref.  The field is always assigned to so it would make more sense.

Reply:
Will look into. I don't remember why I chose ref instead of out for this.